resources/measures/HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -800,6 +881,95 @@ def self.annual_equivalent_full_load_hrs(year_description, schedule)
     return annual_flh
   end
 
+  def self.ruleset_from_fixedinterval(model, hrly_sched, sch_name, winter_design_day_sch, summer_design_day_sch)
+  
+    #Returns schedule rules from a fixed interval object (60 min interval only)    
+    year_description = model.getYearDescription
+    leap_offset = 0
+	  if year_description.isLeapYear
+	    leap_offset = 1
+	  end
+	day_startm = [1, 32, 60 + leap_offset, 91 + leap_offset, 121 + leap_offset, 152 + leap_offset, 182 + leap_offset, 213 + leap_offset, 244 + 	leap_offset, 274 + leap_offset, 305 + leap_offset, 335 + leap_offset]	
```

Comment:
I think you're better off taking advantage of this method defined in constants.rb: https://github.com/NREL/OpenStudio-BuildStock/blob/demand_response/resources/measures/HPXMLtoOpenStudio/resources/constants.rb#L40-L44.